lib/cmd/fh3/appforms/forms/import.js

```diff
@@ -0,0 +1,18 @@
+var url = '/api/v2/appforms/forms/import';
+var upload = require('../../../../utils/request').formsImportZipFileUpload;
+
+module.exports = {
+  'desc': 'Imports a zip file containing definitions for multiple forms, intended to be used with *export*',
+  'examples': [{cmd: 'fhc appforms forms import --file=<path to zip file>', desc: 'Imports forms contained in the zip file'}],
+  'demand': ['file'],
+  'alias': {},
+  'describe': {
+    'file': 'Path to a file'
+  },
+  'url': url,
+  'method': 'post',
```

Comment:
I think specifying the method here is not required when using a custom command (you're doing a post anyway)